mmv1/products/monitoring/MonitoredProject.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: MonitoredProject
+base_url: v1/locations/global/metricsScopes
+create_url: v1/locations/global/metricsScopes/{{metrics_scope}}/projects
+delete_url: v1/locations/global/metricsScopes/{{metrics_scope}}/projects/{{name}}
+self_link: v1/locations/global/metricsScopes/{{metrics_scope}}
```

Comment:
Is this the correct self_link? This seems wrong, but is it being used anywhere?

Reply:
This is correct. Without it the read method tries to use `v1/locations/global/metricsScopes/{{name}}`, which is incorrect. The resource is fetched via list.